### PR TITLE
Anya/create profile for existing user backend

### PR DESCRIPTION
**Pull request by awesomeann**
## Describe your changes
1. Added error handling for creating a Profile as unauth user
image

2. added a method to check whether the user already has a profile and added an error handling for trying to create a second profile.
image

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## What kind of feedback do you need?

E.g. Do you need a quick look or a thorough assessment?

## Checklist:

- [X] I have performed a self-review of my code
- [X] I have recommended where to start and how to proceed with the review

### CODE REVIEW

**Review by Lyndallh — APPROVED** [review]

**Review by shayzimm — APPROVED** [review]

**Review by Lyndallh — CHANGES_REQUESTED** [review]
There has been another branch merged into DEV since this branch was last merged with DEV. This PR will need to be merged with the latest DEV and pushed again (see instructions on slack)

**Comment by blakerach1** [comment]
Hey, for consistency, let's change all of the error dictionaries to include "detail" as the key, instead of "message". I think one of the responses that you've screenshot above are django default... so we may as well go with that approach

**Review by blakerach1 — CHANGES_REQUESTED** [review]
Can you please update the error response dictionary so that the key is "detail" rather than "message"? I will do the same on mine

**Comment by awesomeann** [comment]
> There has been another branch merged into DEV since this branch was last merged with DEV. This PR will need to be merged with the latest DEV and pushed again (see instructions on slack)

where do you check that Lindi?

File: DiversiTech/DiversiTech/settings.py [hunk 1]
```diff
@@ -28,7 +28,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = os.environ.get(
     'DJANGO_DEBUG',
-    'False'
+    'True'
```
Comment (shayzimm) [inline DiversiTech/DiversiTech/settings.py]: Can we change this back to false, pls? :)

**Review by blakerach1 — APPROVED** [review]

**Review by shayzimm — CHANGES_REQUESTED** [review]

**Review by shayzimm — APPROVED** [review]

**Review by Lyndallh — APPROVED** [review]

**Pull request merged**